src/init.cpp

```diff
@@ -1996,6 +1996,7 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
                                               cache_sizes.coins,
                                               /*block_tree_db_in_memory=*/false,
                                               /*coins_db_in_memory=*/false,
+                                              /*dash_dbs_in_memory=*/false,
```

Comment:
nit: so far as `coins_db_in_memory` and `dash_dbs_in_memory` are always equal, consider merging them to a new flag `dbs_in_memory`.
`LoadChainstate` has a bit too many arguments already.

Though, keep `block_tree_db_in_memory` as it is because it will be refactored later in bitcoin/bitcoin#30965